lina-acosta/tarea-4/calculator.js

```diff
@@ -0,0 +1,164 @@
+var operatora;
```

Comment:
@LinaAcosta you should use a EM6 approach please review the use of const, let and var

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/#:~:text=var%20declarations%20are%20globally%20scoped%20or%20function%20scoped%20while%20let,the%20top%20of%20their%20scope.